src/main/scala/uk/gov/hmrc/initprototype/HerokuManager.scala

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.initprototype
+import play.api.libs.json.Json
+import scalaj.http.{Http, HttpResponse}
+
+import scala.concurrent.{ExecutionContext, Future, blocking}
+
+class HerokuManager(implicit config: HerokuConfiguration, ec: ExecutionContext) {
```

Comment:
Is there a reason for this config to be implicit? I couldn't see it being used implicitly.

Reply:
It is used implicitly in the companion object but I guess it doesn't need to be. I can make it explicit.